src/TreeSupport.h

```diff
@@ -103,7 +109,7 @@ class TreeSupport
          * can't be on the model and the path to the buildplate isn't clear),
          * the entire branch needs to be known.
          */
-        Node *const parent;
+        Node * parent;
```

Comment:
Code style: No space between class name and `*` pointer operator.

Reply:
Fixed in a3d7485f